src/current/cockroachcloud/export-metrics.md

```diff
@@ -338,6 +338,67 @@ A subset of CockroachDB metrics require that you explicitly [enable percentiles]
+  1. From the **Overview** page for the Application Insights resource, from the *Connection String*, copy the values for the *InstrumentationKey* and *IngestionEndpoint* to be used in Step 5.
+
+1. Issue the following [Cloud API]({% link cockroachcloud/cloud-api.md %}) command to enable metrics export for your CockroachDB {{ site.data.products.dedicated }} cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request POST \
+      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport/azuremonitor \
+      --header "Authorization: Bearer {secret_key}" \
+      --data '{ "application_id": "{application_id}", "tenant_id": "{tenant_id}", "ingestion_endpoint": "{ingestion_endpoint}", "instrumentation_key": "{instrumentation_key}", "certificate": "{combined_certificate_and_key}" }'
+    ~~~
+
+    Where:
+    - `{cluster_id}` is your CockroachDB {{ site.data.products.dedicated }} cluster ID from step 1.
+    - `{secret_key}` is your CockroachDB {{ site.data.products.dedicated }} API key. See [API Access]({% link cockroachcloud/managing-access.md %}) for instructions on generating this key.
+    - `{application_id}` is the ID of the Entra application, *Application (client) ID* from step 3.
+    - `{tenant_id}` is the ID of the tenant where the Entra application was created, *Directory (tenant) ID* from step 3.
+    - `{ingestion_endpoint}` is the target host where the metrics are to be sent, *IngestionEndpoint* from step 4.
+    - `{instrumentation_key}` is the ID of the Azure Application Insights resource, *InstrumentationKey* from step 4.
+    - `{combined_certificate_and_key}` is the string in the combined file from step 2.
+
+1. Depending on the size of your cluster and how many regions it spans, the configuration may take a moment. You can monitor the ongoing status of the configuration using the following Cloud API command:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request GET \
+      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport/azuremonitor \
+      --header "Authorization: Bearer {secret_key}"
+    ~~~
+
+    Run the command periodically until the command returns a status of `ENABLED`, at which point the configuration across all nodes is complete, and metrics will begin appearing in Azure Monitor. Since the configuration is applied to cluster nodes in a rolling fashion, you may see some metrics appear even before the `GET` command returns an `ENABLED` status.
+
+1. Once metrics export has been enabled, you can access metrics from your CockroachDB {{ site.data.products.dedicated }} cluster directly in [Azure Monitor](https://learn.microsoft.com/en-us/azure/azure-monitor/).
```

Comment:
```suggestion
Once metrics export has been enabled, you can access metrics from your CockroachDB {{ site.data.products.dedicated }} cluster directly in [Azure Monitor](https://learn.microsoft.com/en-us/azure/azure-monitor/).
```
This isn't really a step, but the procedure conclusion.

Reply:
updated